.circleci/config.yml

```diff
@@ -210,7 +210,7 @@ commands:
           command: |
             conda activate kedro-viz
             python -m pip install -U pip setuptools wheel
-            pip install git+https://github.com/quantumblacklabs/kedro@master
+            pip install git+https://github.com/quantumblacklabs/kedro@main
```

Comment:
We are changing from `master` to `main`  for Viz only for now, is that correct?

Reply:
Yeah that's correct, this was a mistake on my part, thanks for spotting!